tensorboard/webapp/tb_debug/BUILD

```diff
@@ -0,0 +1,13 @@
+load("//tensorboard/defs:defs.bzl", "tf_ts_library")
```

Comment:
Do you intend this package to be used by Angular app too? If not, I think it would be confusing to place it here. Move it to tensorboard/components?

Reply:
Done.